server/routes/journeys/develop-an-initial-plan/routes.ts

```diff
@@ -10,8 +10,9 @@ import { RecordActionsProgressRoutes } from './record-actions-progress/routes'
 import { DeleteIdentifiedNeedRoutes } from './delete-identified-need/routes'
 import { NextReviewDateRoutes } from './next-review-date/routes'
 import { InterventionDetailsRoutes } from './intervention-details/routes'
+import { PlanCheckAnswersRoutes } from './check-answers/routes'
 
-function Routes(_services: Services) {
+function Routes(services: Services) {
   const { router, get, post } = JourneyRouter()
```

Comment:
i actually expected the placeholder to be expanded into `Routes({ csipApiService }: Services)`
so the parent route passes around the services as a whole package, but the routes that actually make use of the services can be explicit about which of the services it uses

Reply:
Done